extension/src/test/suite/experiments/model/filterBy/tree.test.ts

```diff
@@ -54,6 +55,7 @@ suite('Experiments Filter By Tree Test Suite', () => {
 
   afterEach(() => {
     disposable.dispose()
+    return closeAllEditors()
```

Comment:
[F] This set of tests opens webviews and should close them when it is done with them.

Reply:
same goes for the other suites where this is added